src/components/accordion-item/daikin-accordion-item.ts

```diff
@@ -0,0 +1,196 @@
+    },
+  }
+);
+
+const animationOption = {
+  duration: 250,
+  easing: "ease-in-out",
+};
+
+const contentCloseKeyframe = {
+  height: 0,
+};
+const getContentOpenKeyframe = (content: HTMLElement) => ({
+  height: `${content.offsetHeight}px`,
+});
+
+/**
+ * Primary UI component for user interaction
+ */
+@customElement("daikin-accordion-item")
+export class DaikinAccordionItem extends LitElement {
+  static override readonly styles = css`
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      display: block;
+      width: 100%;
+      max-width: 400px;
+    }
+  `;
+
+  private _contentRef = createRef<HTMLElement>();
+
+  /**
+   * Heading of accordion
+   */
+  @property({ type: String })
+  override title = "";
+
+  /**
+   * Whether the accordion is open
+   */
+  @property({ type: Boolean, reflect: true })
+  open = false;
+
+  /**
+   * Whether the accordion is disabled
+   */
+  @property({ type: Boolean, reflect: true })
+  disabled = false;
+
+  /**
+   * Open attribute of the actual details element
+   *
+   * The default `open` attribute of the default details element does not allow the display of content to have transitions.
+   * To solve this, the `open` property that `daikin-accordion-item` receives manages the opening and closing of items independently of the open attribute.
+   *
+   * The `open` attribute, which should be present, is taken over by `_detailsOpen`.
+   */
+  @state()
+  private _detailsOpen = false;
+
+  private _handleSummaryClick(e: PointerEvent) {
+    const content = this._contentRef.value;
+
+    e.preventDefault();
+
+    if (this.disabled || !content) {
+      return;
+    }
+
+    if (this.open) {
+      // Accordion is opened; close it.
+      this.open = false;
+      const animation = content.animate(
+        [getContentOpenKeyframe(content), contentCloseKeyframe],
+        animationOption
+      );
+
+      animation.onfinish = () => {
+        // After the animation is finished, remove the open attribute from the details element. This is to allow the element to transition.
+        this._detailsOpen = this.open;
+      };
+    } else {
+      // Accordion is closed; open it.
+      this.open = true;
+      this._detailsOpen = this.open;
+      content.animate(
+        [contentCloseKeyframe, getContentOpenKeyframe(content)],
+        animationOption
+      );
+    }
+  }
+
+  override render() {
+    const accordionDetailsClassName = cvaDetails({
+      status: this.disabled ? "disabled" : "enabled",
+    });
+
+    const accordionSummaryClassName = cvaSummary({
+      status: this.disabled ? "disabled" : "enabled",
+      visible: this.open ? "open" : "close",
+    });
+
+    return html`<details
+      class=${accordionDetailsClassName}
+      ?open=${this._detailsOpen}
+      ?data-open=${this.open}
+      aria-disabled=${this.disabled}
+    >
+      <summary
+        class=${accordionSummaryClassName}
+        @click=${this._handleSummaryClick}
+        tabindex=${!this.disabled ? 0 : -1}
```

Comment:
I mean placing `@click` below `tabindex`.

```suggestion
        tabindex=${this.disabled ? -1 : 0}
        @click=${this._handleSummaryClick}
```